### DISCUSSION TITLE
reflex init  fail!

### DISCUSSION THREAD

**Discussion by xile1973** [reflex-dev/reflex#2438] [Q&A]
/myproj/.venv/lib/python3.10/site-packages/reflex/utils/prerequisites.py", line 615, in install_bun
    raise FileNotFoundError("Reflex requires unzip to be installed.")
FileNotFoundError: Reflex requires unzip to be i

**Comment by picklelo** [comment]
Can you try to install `unzip` on your machine?

**Reply by xile1973** [reply]
Thanks,I install unzip on my machine,reflex is runing now!
